.github/workflows/cve_checks.yml

```diff
@@ -1,5 +1,8 @@
 name: "Infra: CVE checks"
 on:
+  pull_request:
```

Comment:
This one implies specifying types, for example we don't need to re-run this on `labeled` event

Reply:
Updated it with https://github.com/kafbat/kafka-ui/pull/745/commits/fcd6acbd43b38e6978f288537d43bb61ba58c8d2